### PR TITLE
Internal: Upgrade to Node 18 [ED-8708]

### PR DESCRIPTION
**Pull request by yotamselementor**
## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [x] I have added unittests to verify the code works as intended
- [x] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -85,10 +87,10 @@ jobs:
...
           php8.0 vendor/bin/parallel-lint --blame --exclude node_modules --exclude vendor .
           php8.1 vendor/bin/parallel-lint --blame --exclude node_modules --exclude vendor .
-      - name: Setup PHP 7.1 # not included in ubuntu 18.04
-        uses: shivammathur/setup-php@9882bed06691b3a085010c1602ce43ef18f15c5b # v2
-        with:
-          php-version: '7.1'
+      # - name: Setup PHP 7.1 # not included in ubuntu 18.04
+      #   uses: shivammathur/setup-php@9882bed06691b3a085010c1602ce43ef18f15c5b # v2
+      #   with:
+      #     php-version: '7.1'
```
Comment (StyleShit) [inline .github/workflows/lint.yml:93]: remove the comment and add the linting to the list above

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -55,10 +55,10 @@ jobs:
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
```
Comment (StyleShit) [inline .github/workflows/playwright.yml:57]: v3?
also update other actions if there are

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: .github/workflows/playwright.yml (hunk 2 above)
Comment (yotamselementor) [reply .github/workflows/playwright.yml]: https://github.com/elementor/elementor/pull/20377/commits/eccd29e265e7eb8ee0138555b0d26ac71019e025

File: .github/workflows/lint.yml (hunk 1 above)
Comment (yotamselementor) [reply .github/workflows/lint.yml]: https://github.com/elementor/elementor/pull/20377/commits/eccd29e265e7eb8ee0138555b0d26ac71019e025

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -9,40 +9,42 @@ concurrency:
...
+            .github/**/*.yml
       - name: Check PHP files diff
         id: php_diff_files
-        uses: technote-space/get-diff-action@v4
+        uses: technote-space/get-diff-action@v6
         with:
           PATTERNS: |
             **/*.php
-
+            .github/**/*.yml
```
Comment (StyleShit) [inline .github/workflows/lint.yml:35]: composer?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml (hunk 3 above)
Comment (yotamselementor) [reply .github/workflows/lint.yml:35]: https://github.com/elementor/elementor/pull/20377/commits/854b2b90430531e26508f521feefb24bf584d2c6

**Review by StyleShit — APPROVED** [review]

**Pull request merged**